snooty/parser.py

```diff
@@ -712,6 +713,19 @@ def handle_directive(
                 )
                 return doc
 
+        elif name == "openapi-changelog":
+            # Version Changelog will be dependent on present api-version option
+            api_version = options.get("api-version", None)
+
+            if not argument_text == "cloud":
+                self.diagnostics.append(ExpectedPathArg(name, line))
+                return doc
+
+            if api_version:
+                return doc
+            else:
```

Comment:
No need to put this in an `else`, since if the condition is true we immediately return.